hebrewreader.py

```diff
@@ -22,23 +22,25 @@
 )
 
 FEATURES = 'g_word_utf8 gloss lex_utf8 otype trailer_utf8 voc_lex_utf8'
+SYR_FEATURES = 'otype trailer word'
+GRK_FEATURES = 'g_cons_utf8 gloss lex_utf8 otype word'
 
 VERSE_NODES = dict()
 
 def load_verse_nodes():
     global VERSE_NODES
-
```

Comment:
Throughout this file, please undo unnecessary changes (removed newlines, `print` statements).